stdlib/universe/order_filter_group_drop_empty_flag_off_test.flux

```diff
@@ -0,0 +1,59 @@
+package universe_test
+
+import "testing"
+import "planner"
+
+option now = () => (2030-01-01T00:00:00Z)
+option planner.disableLogicalRules = ["GroupFilterTransposeRule"]
+
+input = "
+#datatype,string,long,string,string,dateTime:RFC3339,double
+#group,false,false,true,true,false,false
+#default,_result,0,Wx8gaJ,U2FstZ,,
+,result,table,foo,bar,_time,_value
+
+#datatype,string,long,string,string,dateTime:RFC3339,double
+#group,false,false,true,true,false,false
+#default,_result,1,U37BFq3,KYxg,,
+,result,table,foo,bar,_time,_value
+
+#datatype,string,long,string,string,dateTime:RFC3339,double
+#group,false,false,true,true,false,false
+#default,_result,2,U37BFq3,U2FstZ,,
+,result,table,foo,bar,_time,_value
+
+#datatype,string,long,string,string,dateTime:RFC3339,double
+#group,false,false,true,true,false,false
+#default,_result,3,Wx8gaJ,KYxg,,
+,result,table,foo,bar,_time,_value
+"
+
+output = "
+#datatype,string,long,string,string,dateTime:RFC3339,double
+#group,false,false,true,true,false,false
+#default,_result,0,Wx8gaJ,U2FstZ,,
+,result,table,foo,bar,_time,_value
+
+#datatype,string,long,string,string,dateTime:RFC3339,double
+#group,false,false,true,true,false,false
+#default,_result,1,U37BFq3,KYxg,,
+,result,table,foo,bar,_time,_value
+
+#datatype,string,long,string,string,dateTime:RFC3339,double
+#group,false,false,true,true,false,false
+#default,_result,2,U37BFq3,U2FstZ,,
+,result,table,foo,bar,_time,_value
+
+#datatype,string,long,string,string,dateTime:RFC3339,double
+#group,false,false,true,true,false,false
+#default,_result,3,Wx8gaJ,KYxg,,
+,result,table,foo,bar,_time,_value
+"
+
+order_fn = (tables=<-) => tables
+    |> range(start: 2018-05-22T19:53:26Z)
+    |> group(columns: ["_time"])
+    |> filter(fn: (r) => r["_field"] == "load4", onEmpty: "drop")
```

Comment:
Could you update this query to be a bit more meaningful? 
- You could group by the column `foo` or `bar` so that two tables are produced
- For the filter, none of the tables have a `_field` set to `"load4"` so that doesn't really make sense. You could instead filter on `r.bar == "Wx8gaJ"` or something like that?

(Note that even if there are no rows in the table, the group key columns still have values.

Reply:
(this also goes for the other tests)